.github/workflows/test-r-scripts.yaml

```diff
@@ -63,13 +76,27 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
+      - name: Install system dependency of usethis >= 2.0.0
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt install libgit2-dev
+
       - name: Install dependencies from CRAN
         run: |
           install.packages("rmarkdown")
           install.packages("knitr")
```

Comment:
It's been a while since I did this but this is what I think I remember:

If knitr is installed, then renv can detect dependencies in .Rmd files, and rmarkdown is a dependency of knitr).

Reply:
Here it is 

https://rstudio.github.io/renv/reference/dependencies.html#details

> Note that the rmarkdown package is required in order to crawl dependencies in R Markdown files.

Reply:
👍🏻 thanks, good to know